README.rst

```diff
@@ -374,6 +378,52 @@ running the doctests with sphinx is not supported. To do this, add
 ``conf.py`` file.
 
 
+Fixing Existing Docstrings
+--------------------------
+The plugin has basic support to fix docstrings, this can be enabled by
+running `pytest` with `--doctest-plus-generate-diff`.
+Without further options, this will print out a diff and a list of files that
+would be modified.  Using `--doctest-plus-generate-diff=overwrite` will
+modify the files in-place, so it is recommended to run the check first to
+verify the paths.
+You may wish to use e.g. `git commit -p` to review changes manually.
+
+The current diff generation is not very smart, so it does not account for
+existing `...`.  By default a diff is only generated for *failing* doctests.
+
+In general, a mass edit may wish to focus on a specific change and
+possibly include passing tests.  So you can hook into the behavior by
+adding a hook to your `conftest.py`:
+```python
+@pytest.hookimpl
+def pytest_doctestplus_diffhook(info):
+    info["use"] = True  # Overwrite all results (even successes)
+    if info["fileno"] is None:
+        # E.g. NumPy has C docstrings that cannot be found, we can add custom
+        # logic here to try and find these:
+        info["filename"] = ...
+        info["lineno"] = ...
+```
+Where `info` is a dictionary containing the following items:
+* `use`: `True` or `False` signalling whether to apply the diff.  This is
```

Comment:
Some whitespacing and double backticking will be needed here to make this section render well.
```suggestion
adding a hook to your `conftest.py`:
```python
@pytest.hookimpl
def pytest_doctestplus_diffhook(info):
    info["use"] = True  # Overwrite all results (even successes)
    if info["fileno"] is None:
        # E.g. NumPy has C docstrings that cannot be found, we can add custom
        # logic here to try and find these:
        info["filename"] = ...
        info["lineno"] = ...
```
Where `info` is a dictionary containing the following items:
* `use`: `True` or `False` signalling whether to apply the diff.  This is
```

Reply:
Sorry, rst, not md :).